cs-amazon/src/main/java/io/cloudslang/content/amazon/utils/OutputsUtil.java

```diff
@@ -47,19 +47,19 @@ private OutputsUtil() {
 
     public static void putResponseIn(Map<String, String> queryMapResult) {
         if (queryMapResult != null) {
-            if (queryMapResult.containsKey(STATUS_CODE) && (valueOf(SC_OK).equals(queryMapResult.get(STATUS_CODE))) && queryMapResult.containsKey(RETURN_RESULT) && !isBlank(queryMapResult.get(RETURN_RESULT))) {
-                queryMapResult.put(RETURN_CODE, SUCCESS);
-            } else {
-                queryMapResult.put(RETURN_CODE, FAILURE);
-            }
+            setReturnCode(queryMapResult);
         } else {
-            queryMapResult = new HashMap<>();
-            queryMapResult.put(EXCEPTION, "Null response!");
-            queryMapResult.put(RETURN_CODE, FAILURE);
-            queryMapResult.put(RETURN_RESULT, "The query returned null response!");
+            setNullResponseResults();
```

Comment:
@Tirla-Alin In the OutputUtil class there are some problems related to putResponseIn method. Find Bugs and Inspect Code tools both reported it ... So, I think is better to refactor yourself since you know the logic behind it. Personal I don't agree with the existing approach. I will revert to the previous state the above mentioned class.